pootle/apps/pootle_project/models.py

```diff
@@ -56,6 +59,20 @@
 
 class ProjectManager(models.Manager):
 
+    def create(self, *args, **kwargs):
```

Comment:
I raised a question re porting filetypes logic to ProjectManager.create(). And I understood @phlax was going to check this.